migrator/migrations/m_105_to_m_106_group_id/migration_test.go

```diff
@@ -0,0 +1,150 @@
+package m105tom106
+
+import (
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/bolthelpers"
+	"github.com/stackrox/rox/pkg/testutils"
+	"github.com/stretchr/testify/suite"
+	bolt "go.etcd.io/bbolt"
+)
+
+func TestMigration(t *testing.T) {
+	suite.Run(t, new(migrateServiceIdentitySerial))
+}
+
+type migrateServiceIdentitySerial struct {
+	suite.Suite
+
+	db *bolt.DB
+}
+
+func (suite *migrateServiceIdentitySerial) SetupTest() {
+	db, err := bolthelpers.NewTemp(testutils.DBFileName(suite))
+	suite.Require().NoError(err, "failed to make BoltDB")
+	suite.db = db
+}
+
+func (suite *migrateServiceIdentitySerial) TearDownTest() {
+	testutils.TearDownDB(suite.db)
+}
+
+func (suite *migrateServiceIdentitySerial) TestMigrate() {
+	// Expected groups after migration.
+	// Group "r1" should not be updated, as the ID is pre-existing.
+	// Group "r2" should have the ID after migration.
+	// Group "r3" should be migrated but not have an updated ID.
```

Comment:
```suggestion
	// Expected groups after migration. Note that:
	// * Group "r1" should not be updated, as it is stored by ID,
	// * Group "r2" should get an ID after migration,
	// * Group "r3" should be migrated despite it has an ID because it is stored by composite key.
```

Groups "r1" and "r3" look identical modulo role name and hence is unclear why one should be migrated and one should not.